web/src/pages/billing/Billing.css

```diff
@@ -0,0 +1,15 @@
+#group-by-dropdown .menu {
```

Comment:
Can we collapse this into the `index.css` stylesheet?

Reply:
We can, but it might be nicer/cleaner to keep them separate?